dbld/images/centos7/Dockerfile

```diff
@@ -3,11 +3,13 @@ MAINTAINER Andras Mitzki <andras.mitzki@balabit.com>
 
 RUN yum install -y wget epel-release
 
-RUN cd /etc/yum.repos.d && wget https://copr.fedorainfracloud.org/coprs/czanik/syslog-ng39/repo/epel-7/czanik-syslog-ng39-epel-7.repo
+RUN cd /etc/yum.repos.d && wget https://copr.fedorainfracloud.org/coprs/czanik/syslog-ng311/repo/epel-7/czanik-syslog-ng311-epel-7.repo
 
 ADD dev-dependencies.txt .
 RUN cat dev-dependencies.txt | grep -v "#" | xargs yum -y  install
-
+ADD gradle_installer.sh .
+RUN ./gradle_installer.sh
+RUN find / -name libjvm.so | xargs dirname | tee --append /etc/ld.so.conf.d/openjdk-libjvm.conf && ldconfig
```

Comment:
You can cheat :) While the directory name is not fix, there is a symlink to it which is fix. So, as you can see in https://www.balabit.com/blog/syslog-ng-and-elasticsearch-5-getting-started-on-rhelcentos/ you can use a hard coded path name:
```
echo /usr/lib/jvm/jre/lib/amd64/server > /etc/ld.so.conf.d/java.conf
ldconfig
```
This will work as long as you work on AMD64